frame/support/test/tests/pallet.rs

```diff
@@ -504,6 +513,11 @@ pub mod pallet2 {
 		}
 		fn on_runtime_upgrade() -> Weight {
 			Self::deposit_event(Event::Something(31));
+
+			if dbg!(UpdateStorageVersion::get()) {
+				dbg!(Self::current_storage_version()).put::<Self>();
+			}
+
```

Comment:
```suggestion
```

Reply:
That breaks the test :P But yeah, the dbg is wrong.